atom/browser/api/atom_api_session.h

```diff
@@ -81,6 +82,9 @@ class Session: public mate::TrackableObject<Session>,
   void GetBlobData(const std::string& uuid,
                    const AtomBlobReader::CompletionCallback& callback);
   void CreateInterruptedDownload(const mate::Dictionary& options);
+  void AddPreload(const base::FilePath::StringType& preloadPath);
+  void RemovePreload(const base::FilePath::StringType& preloadPath);
+  std::vector<base::FilePath::StringType> GetPreloads();
```

Comment:
Could you use `const std::vector<base::FilePath::StringType>&` to avoid copying.